changelogs/fragments/81494-remove-duplicated-file-attribute-constant.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - "Remove duplicated and unused FILE_ATTRIBUTES constant in module_utils/basic.py file."
```

Comment:
To me this indicates that the constant cannot be imported from `module_utils/basic.py` anymore which isn't true. Although the constant was removed, it was replaced with identical constant from another file and can still be used in 3rd party code. The changelog should mention that.

Reply:
That does make sense. How about I leave out the `module_utils/basic.py` part and write:

```python
bugfixes:
  - "Remove duplicated and unused FILE_ATTRIBUTES constant."
```

Reply:
I don't think it would hurt to be specific:
```yaml
bugfixes:
  - Import ``FILE_ATTRIBUTES`` from ``ansible.module_utils.common.file`` in ``ansible.module_utils.basic`` instead of defining it twice.
```

Reply:
Fair enough, done.